docs/ldml/tr35.md

```diff
@@ -1721,9 +1730,8 @@ There may be specific exceptions to these for certain closely-related languages
 
 There are certain invariants that must always be true:
 
-3. The parent must either be the root locale or have the same script as the child.
```

Comment:
This requirement is still true for the non-component parentLocales data. And for the component ones the only cross-script parents are related, i.e. both Hans and Hant are subsets of Hani, so we only allow inheritance across scripts when they are such subsets.

Reply:
Collator has an inheritance in Serbian that breaks this rule

Reply:
Ah OK. But we can still keep this condition for the non-component parentLocales.

Reply:
Restored the rule.